### ISSUE TITLE
Quewstion about to test a scenario

### ISSUE THREAD

**Issue by Rass1968** [NPBruce/valkyrie#524]
Should it work if I´m clicking through a scenario only in one investigator phase apart from some events f.e. a monster is used?
I ask because I run into the following situation in "House of Lynch" German version:
I clicked through the first markers at the beginning. The one who leads to the bedroom say I need the silver key wich I didn´t had at this point. Right after I clicked it the second time and I could open it!?

**Comment by LordPyrex** [comment]
Some of the written events will tell you that you need a key to proceed, and then on a second click it will give you the exploration option while the dialog text will tell you that you need to have a specific item in order to explore.

I know which door you are talking about. A second click on the Explore Token will generate a box that says something to the effect of you need the silver key to continue. So while you could open it if you just want to click it, in gameplay you wouldn't be able to unless you had the silver key in your possession.

**Comment by Rass1968** [comment]
I started the scenario, clicked on the token, clicked a second time and it opened without the key. Don´t understand what should be different to gameplay?

**Comment by LordPyrex** [comment]
I don't know if there was a text error, but the only example I can find of this in the scenario was something that plays out like this.

![stepone](https://user-images.githubusercontent.com/27811809/27092260-b5968970-5030-11e7-9021-ddab3fa2b903.png)
You see the door
![steptwo](https://user-images.githubusercontent.com/27811809/27092267-bff9e786-5030-11e7-9093-a7c782a37878.png)
Find it locked
![stepthree](https://user-images.githubusercontent.com/27811809/27092287-c8531100-5030-11e7-87c8-c1c255016a8e.png)
The next time that you click the explore token it tells you that you will need a key to continue through this door.

With the gameplay mechanics an Investigator wouldn't be able to interact with that door until they found the key. The randomness that this scenario uses for item placement makes it easier to use this method of interaction with that door. While you can click through it even if you don't possess the key, it would be a bit of a cheat.

Let me know if you were referring to something else though. I can't read German, so I don't know if there is a translation problem.

**Comment by Rass1968** [comment]
Thx for the explanation in detail, understand now the way it is.
But I don´t understand why this way is chosen, either you can open it or not. Or is it because the silver key can "spawn" in that room? confused ;)
It should be the way like the Garden door is. That you can not open till you have the sigil (At the garden door are translation problems as written in the translation thread)

**Comment by LordPyrex** [comment]
The key would not spawn in that room because of the way the Token pools handle found items. This would be part of a pool of items that only spawn outside of that room. I'd have to let Bruce answer you as to the reason that interaction is set differently.

**Comment by NPBruce** [comment]
The reason it has to ask is I don't know which investigator is openning the door, only the one with the key is able to do so.  The message should change once the key has been found, if it changes before the key is found that is a bug, I will investigate.

**Comment by LordPyrex** [comment]
@NPBruce  If there is a bug it looks like it might be related to EventMasterLock. It seems like it removes the starting door token regardless of whether or not the key is found.

**Comment by NPBruce** [comment]
That is correct - once you have explored once the token changes because then you know that it needs the key.

It shouldn't be possible to open it until you have the key though.

EventMasterTile needs to be split into 2, one of which requires clue1.
